src/mame/rm/rm380z.h

```diff
@@ -121,8 +134,13 @@ class rm380z_state : public driver_device
 	DECLARE_MACHINE_RESET(rm480z);
 
 	void config_memory_map();
-	void update_screen_vdu80(bitmap_ind16 &bitmap);
-	void update_screen_vdu40(bitmap_ind16 &bitmap);
+	void palette_init(palette_device &palette) const;
+	void change_hrg_scratchpad(int index, uint8_t value, uint8_t mask);
+	void change_palette(int index, uint8_t value) const;
```

Comment:
These are also conceptually not the sort of things that should be `const`.

Reply:
Yes, I agree about these so have changed the palette altering methods to be non const.